src/main/java/org/team1540/robot2019/utils/ControlUtils.java

```diff
@@ -21,4 +21,33 @@ public static double simpleDeadzone(double input, double deadzone) {
     public static double allVelocityConstraints(double output, double max, double min, double deadzone) {
         return simpleDeadzone(velocityPosNegConstrain(output, max, min), deadzone);
     }
+
+    /**
+     * https://www.desmos.com/calculator/ybuyhcfzgm
```

Comment:
A tad unorthodox documentation-wise but uh I'll allow it

Reply:
hey, this is high-quality documentation